lxd/daemon_images.go

```diff
@@ -26,6 +26,7 @@ import (
 // Simplestream cache
 type imageStreamCacheEntry struct {
```

Comment:
I still find `image stream` terminology confusing. `channel` or `source` sounds better, because images are not streamed in real time from remote server, but just stored there and updated periodically.